manifests/0000_50_olm_05-subscription.crd.yaml

```diff
@@ -181,4 +181,1282 @@ spec:
                           optional:
                             type: boolean
                             description: Specify whether the Secret must be defined
-
+                volumeMounts:
+                  description: List of VolumeMounts to set in the container.
+                  items:
+                    description: VolumeMount describes a mounting of a Volume within
+                      a container.
+                    properties:
+                      mountPath:
+                        description: Path within the container at which the volume
+                          should be mounted.  Must not contain ':'.
+                        type: string
+                      mountPropagation:
+                        description: mountPropagation determines how mounts are
+                          propagated from the host to container and the other way
+                          around. When not set, MountPropagationNone is used. This
+                          field is beta in 1.10.
+                        type: string
+                      name:
+                        description: This must match the Name of a Volume.
+                        type: string
+                      readOnly:
+                        description: Mounted read-only if true, read-write otherwise
+                          (false or unspecified). Defaults to false.
+                        type: boolean
+                      subPath:
+                        description: Path within the volume from which the container's
+                          volume should be mounted. Defaults to "" (volume's root).
+                        type: string
+                      subPathExpr:
+                        description: Expanded path within the volume from which
+                          the container's volume should be mounted. Behaves similarly
+                          to SubPath but environment variable references $(VAR_NAME)
+                          are expanded using the container's environment. Defaults
+                          to "" (volume's root). SubPathExpr and SubPath are mutually
+                          exclusive. This field is beta in 1.15.
+                        type: string
+                    required:
+                      - mountPath
+                      - name
+                    type: object
+                  type: array
+                volumes:
+                  description: List of Volumes to set in the podSpec.
+                  items:
+                    description: Volume represents a named volume in a pod that
+                      may be accessed by any container in the pod.
+                    properties:
+                      awsElasticBlockStore:
+                        description: 'AWSElasticBlockStore represents an AWS Disk
+                                          resource that is attached to a kubelet''s host machine
```

Comment:
YAML lint is complaining this is not valid YAML file due to double 'single quote' on this line.

Reply:
The error you mentioned above seems to be specific to the linter. Ultimately, the CRD is accepted by k8s so this shouldn't be an issue.